.github/workflows/main.yml

```diff
@@ -0,0 +1,27 @@
+on: ["push", "pull_request"]
+name: Test
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.13.x, 1.14.x]
```

Comment:
Is there a `go-latest` I wonder?

Reply:
nop :disappointed:   - see https://github.com/actions/setup-go/issues/21